packages/react/src/provider.tsx

```diff
@@ -10,6 +10,7 @@ export const NhostReactContext = createContext<NhostClient>({} as NhostClient)
 export const NhostReactProvider: React.FC<{
```

Comment:
My suggestion would be to expose the interface of `NhostReactProvider` and use `PropsWithChildren` type to add children to the list of props.

```diff
- export const NhostReactProvider: React.FC<{
-   nhost: NhostClient,
-   initial?: NhostSession,
-   children?: React.ReactNode
- }> = /* ... */
+ export interface NhostReactProviderProps {
+   nhost: NhostClient
+   initial?: NhostSession
+ }
+
+ export function NhostReactProvider({
+   nhost,
+   initial,
+   ...props
+ }: PropsWithChildren<NhostReactProviderProps>) { /*...*/ }
```

Reply:
thanks for the suggestion! done.